composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "beans/beans",
+    "description": "The most innovative theme for WordPress. The real magic is under the hood.",
+    "type": "wordpress-theme",
+    "license": "GPL-2.0+",
+    "homepage": "http://www.getbeans.io/",
```

Comment:
This website seems to redirect `https://` to the `http://` site, which seems backwards.

Reply:
I’ll add that to my list of things to fix. Thank you.